### DISCUSSION TITLE
[FUNDAMENTOS C# .NET] Versões .NET

### DISCUSSION THREAD

**Discussion by luanhenriquee** [balta-io/forum#430] [Tira Dúvidas] [answered]
Como posso excluir uma versão do framework, ou decidir qual versão irei usar em X projeto?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @luanhenriquee, como vai?

Se for a versão do .NET, não precisa excluir, só instalar e pronto. Elas roda lado a lado.

Para alterar a versão do .NET usada no seu projeto, basta editar o arquivo `.csproj` e trocar o `TargetFramework`.
```
<TargetFramework>net8.0</TargetFramework>
```

Para alterar a versão de um pacote, você precisa primeiro removê-lo:
```
dotnet remove package NOME_DO_PACOTE
```

E depois adicionar a versão específica:
```
dotnet add package NOME_DO_PACOTE --version X.X.X

Exemplo:
dotnet add package Flunt --version 2.0.5
```

**Reply by luanhenriquee** [reply]
Só para finalizar, posso ter diversas versões do .NET em minha máquina, e para eu alterar a versão usada no projeto é só alterar a `.csproj`?

**Reply by oimarcella** [reply]
Sim, pode ter diversas versões do .net na sua máquina, elas não conflitam, rodam lado a lado. E sobre o projeto, é isso mesmo que você disse.
